src/raster/r.area.createweight/r.area.createweight.html

```diff
@@ -224,18 +224,23 @@ <h4>Python libraries</h4>
 For <b>MS-Windows</b> users using a 64 bit GRASS, the easiest way is to use the
 <a href="https://grass.osgeo.org/download/software/ms-windows/">OSGeo4W</a>
 installation method of GRASS, where the Python setuptools can also be
-installed. The users can then run 'easy_install pip' to install the pip
+installed. The users can then run <code>easy_install pip</code> to install the pip
 package manager. Then, they can download the "Python wheels"
 corresponding to each package and install them using the following
-command: <em>pip install packagename.whl</em>. Links for
+command: <code>pip install packagename.whl</code>. Links for
 downloading wheels are provided below. The version installed should be
 compatible with user's Python version. If GRASS was not installed
 using the OSGeo4W method, the pip package manager can be installed by
 saving the "get-pip.py" python script provided
 <a href="https://bootstrap.pypa.io/get-pip.py">here</a> in the folder
 containing the GRASS GIS Python environment
 (GRASSFOLDER/../etc/python/grass) and executing it with administrator
-rights with the following command: <em>python get-pip.py</em>
+rights with the following command: <code>python get-pip.py</code>
+<br>
+For users of <b>macOS</b> GRASS-X.X.app bundles, start GRASS and install
+packages with following commands in the Terminal: <code>python
+-m pip install --upgrade pip</code> and <code>python -m pip
+install pandas scikit-learn</code>.

```

Comment:
```suggestion
packages with following commands in the Terminal:<br>
<code>python -m pip install --upgrade pip</code> and<br>
<code>python -m pip install pandas scikit-learn</code> .
```

I suggest to break the lines here for better readability.

Reply:
I don't think it's necessary to break for the Mac commands only, there are also Win commands.

This is what it will look like as is:

<img width="840" alt="01" src="https://user-images.githubusercontent.com/14186207/149229929-1112194a-fe45-4b34-a05d-b3802fe6d710.png">

with your suggestion:

<img width="840" alt="02" src="https://user-images.githubusercontent.com/14186207/149229969-8eef6391-8ddc-4e51-986e-261ff325424b.png">

I plan to suggest in another PR (in core) css changes to the effect of:

<img width="840" alt="03" src="https://user-images.githubusercontent.com/14186207/149230078-38f8a921-45b7-4ec6-a61f-e1adfb389164.png">

Reply:
OK, fine for me.